deploy-as-code/helm/charts/digit-works/backend/muster-roll-service/values.yaml

```diff
@@ -77,6 +77,10 @@ env: |
     value: "save-musterroll"
   - name: MUSTERROLL_KAFKA_UPDATE_TOPIC
     value: "update-musterroll"
+  {{- if index .Values "app-timezone" }}
```

Comment:
As per key name (i.e : timezone) change the corresponding condition here

Reply:
Updated the values.yaml to read from the key timezone